### PR TITLE
Bug fixes, disable crons and optional MAINTAINERS file

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

File: metainvenio/cli/travis.py [hunk 1]
```diff
@@ -154,3 +154,31 @@ def travis_cron_enable(ctx):
...
+
+@travis.command('cron-disable')
+@click.pass_context
+def travis_cron_disable(ctx):
+    """Enable cron for repositories."""
+    conf = ctx.obj['config']
+    travis = ctx.obj['client']
+
+    for repo in conf.repositories:
+        if repo.travis.active:
```
Comment (david-caro) [inline metainvenio/cli/travis.py]: I would rephrase as:
```python
    for repo in conf.repositories:
        if not repo.travis.active:
            continue

        if travis.cron_disable(repo.slug):
    ...
```

to avoid the extra indent.

File: metainvenio/cli/github.py [hunk 2]
```diff
@@ -59,8 +60,10 @@ def github_repo_configure(ctx):
             click.echo('Updated maintainer team')
         if repoapi.update_branch_protection():
             click.echo('Updated branch protection')
-        if repoapi.update_maintainers_file():
-            click.echo('Updated MAINTAINERS file')
+        if with_maintainers_file:
+            click.echo('Checking MAINTAINERS file')
+            if repoapi.update_maintainers_file():
```
Comment (david-caro) [inline metainvenio/cli/github.py]: If `update_maintainers_file` returns `False` it gets ignored? Is that ok? Or should it show another message like `'MAINTAINERS file already ok'`?

File: metainvenio/travis.py [hunk 3]
```diff
@@ -125,6 +125,14 @@ def cron_enable(self, repo_slug, branch, interval='weekly',
         })
         return get_response_contents(res)
 
+    def cron_disable(self, repo_slug):
+        """Disable all cron jobs for a repository."""
+        for cron in self.crons(repo_slug):
+            res = self.v3client.delete('{}/cron/{}'.format(V3URI, cron.id))
+            if res.status_code != 204:
+                return False
```
Comment (david-caro) [inline metainvenio/travis.py]: I'm not sure if it already does, but what happens on non 2XX return codes? (5XX, 4XX specially)
I think it should rise an exception in those cases.

Comment (david-caro) [inline metainvenio/travis.py]: Or at least show an error message so the user knows that something bad happened.

File: metainvenio/cli/github.py (hunk 2 above)
Comment (lnielsen) [inline metainvenio/cli/github.py]: It's similar to the other parts above, where only if the branch protection was updated it will print some info (i.e. lack of output means that everything was fine).

File: metainvenio/travis.py (hunk 3 above)
Comment (lnielsen) [inline metainvenio/travis.py]: The CLI will display an error message saying [``Failed to disable crons for <repo>.``](https://github.com/inveniosoftware/metainvenio/pull/11/files#diff-1709947950917af590ffe54704b7f0a3R173) and then continue with disabling cron for the next repo. Does that work?

Comment (david-caro) [inline metainvenio/travis.py]: 👍
